cleanco/termdata.py

```diff
@@ -1,144 +1,131 @@
+    'Iceland': ['ehf.', 'hf.', 'ohf.', 's.f.', 'ses.'],
+    'India': ['pvt. ltd.', 'ltd.', 'psu', 'pse'],
+    'Indonesia': ['ud', 'fa', 'pt'],
+    'Ireland': ['cpt', 'teo'],
+    'Israel': ['b.m.', 'bm', 'ltd', 'limited'],
+    'Italy': ['s.n.c.', 's.a.s.', 's.p.a.', 's.a.p.a.', 's.r.l.', 's.c.r.l.', 's.s.'],
+    'Latvia': ['as', 'sia', 'ik', 'ps', 'ks'],
+    'Lebanon': ['sal'],
+    'Lithuania': ['uab', 'ab', 'ij', 'mb'],
+    'Luxemborg': ['s.a.', 's.a.r.l.', 's.a r.l.', 'secs'],
+    'Macedonia': ['d.o.o.', 'd.o.o.e.l', 'k.d.a.', 'j.t.d.', 'a.d.', 'k.d.'],
+    'Malaysia': ['bhd.', 'sdn. bhd.'],
+    'Mexico': ['s.a.', 's. de. r.l.', 's. en c.', 's.a.b.', 's.a.p.i.'],
+    'Mongolia': ['xk', 'xxk'],
+    'Netherlands': ['v.o.f.', 'c.v.', 'b.v.', 'n.v.'],
+    'New Zealand': ['tapui', 'ltd', 'limited'],
+    'Nigeria': ['gte.', 'plc', 'ltd.', 'ultd.'],
+    'Norway': ['asa', 'as', 'ans', 'ba', 'bl', 'da', 'etat', 'fkf', 'hf', 'iks', 'kf',
+               'ks', 'nuf', 'rhf', 'sf'],
+    'Oman': ['saog', 'saoc'],
+    'Pakistan': ['ltd.', 'pvt. ltd.', 'ltd', 'limited'],
+    'Peru': ['sa', 's.a.', 's.a.a.'],
+    'Philippines': ['coop.', 'corp.', 'corp', 'ent.', 'inc.', 'inc', 'llc', 'l.l.c.',
+                    'ltd.'],
+    'Poland': ['p.p.', 's.k.a.', 'sp.j.', 'sp.k.', 'sp.p.', 'sp. z.o.o.', 's.c.', 's.a.'],
+    'Portugal': ['lda.', 'crl', 's.a.', 's.f.', 'sgps'],
+    'Romania': ['s.c.a.', 's.c.s.', 's.n.c.', 's.r.l.', 'o.n.g.', 's.a.'],
+    'Russia': ['ooo', 'oao', 'zao', '3ao', 'пао', 'оао', 'ооо'],
+    'Serbia': ['d.o.o.', 'a.d.', 'k.d.', 'o.d.'],
+    'Singapore': ['bhd', 'pte ltd', 'sdn bhd', 'llp', 'l.l.p.', 'ltd.', 'pte', 'pte. ltd.'],
+    'Slovenia': ['d.d.', 'd.o.o.', 'd.n.o.', 'k.d.', 's.p.'],
+    'Slovakia': ['a.s.', 'akc. spol.', 's.r.o.', 'spol. s r.o.', 'k.s.', 'kom. spol.', 'v.o.s.', 'a spol.'],
+    'Spain': ['s.a.', 's.a.d.', 's.l.', 's.l.l.', 's.l.n.e.', 's.c.', 's.cra', 's.coop',
+              'sal', 'sccl'],
+    'Sweden': ['ab', 'hb', 'kb'],
+    'Switzerland': ['ab', 'sa', 'gmbh', 'g.m.b.h.', 'sarl', 'sagl'],
+    'Turkey': ['koop.'],
+    'Ukraine': ['dat', 'fop', 'kt', 'pt', 'tdv', 'tov', 'pp', 'vat', 'zat', 'at'],
+    'United Kingdom': ['plc.', 'plc', 'cic', 'cio', 'l.l.p.', 'llp', 'l.p.', 'lp', 'ltd.',
+                       'ltd', 'limited'],
+    'United States of America': ['llc', 'inc.', 'corporation', 'incorporated', 'company',
+                                 'limited', 'corp.', 'inc.', 'inc', 'llp', 'l.l.p.', 'pllc', 'and company',
+                                 '& company', 'inc', 'inc.', 'corp.', 'corp', 'ltd.', 'ltd', '& co.', '& co', 'co.',
+                                 'co', 'lp'],
+    'Uzbekistan': ['mchj', 'qmj', 'aj', 'oaj', 'yoaj', 'xk', 'xt', 'ok', 'uk', 'qk']
+}
+
+
+country_name_by_country = {
+    'Finland': ['finland', 'suomi'],
+    'Sweden': ['sweden', 'sverige'],
+    'Norway': ['norway', 'norge'],
+    'Luxemborg': ['luxemborg', 'luxembourg']
 }
```

Comment:
Worth adding USA too?

Reply:
Surprisingly I found that companies in the US rarely put the country in the name. Probably because it is considered the "main" branch. I think it would lead to more false positives than desired results.